packages/styles/scss/components/card-link/index.scss

```diff
@@ -23,6 +23,10 @@
       margin-bottom: $spacing-07;
     }
 
+    &__image {
+      height: 192px;
+    }
+
```

Comment:
@mkothur are we fixing height like this?
cc: @photodow

Reply:
sorry, now I changed to cards-with-images-group.scss file

Reply:
Trying to define this more in #768. Not sure if this will be defined in this pattern, or not and maybe more based on ratio. We'll see, this should be fine for this for the moment.